packages/manager/src/features/Linodes/LinodesDetail/LinodeNetworking/LinodeInterfaces/EditInterfaceDrawer/VPCInterface/VPCIPv4Address.tsx

```diff
@@ -43,20 +46,36 @@ export const VPCIPv4Address = (props: Props) => {
           name={`vpc.ipv4.addresses.${index}.address`}
           render={({ field, fieldState }) => (
             <Stack rowGap={1}>
-              <FormControlLabel
-                checked={field.value === 'auto'}
-                control={<Checkbox />}
-                label="Auto-assign a VPC IPv4 Address"
-                onChange={(e, checked) =>
-                  field.onChange(
-                    checked
-                      ? 'auto'
-                      : (linodeInterface.vpc?.ipv4?.addresses[index].address ??
-                          '')
-                  )
-                }
-                sx={{ pl: 0.3 }}
-              />
+              <Stack direction="row">
+                <FormControlLabel
+                  checked={field.value === 'auto'}
+                  control={<Checkbox />}
+                  label="Auto-assign a VPC IPv4 Address"
```

Comment:
```suggestion
                  label="Auto-assign VPC IPv4 address"
```

(match Assign Linodes drawer)